src/containers/Dashboard/Forms/CourseForm.js

```diff
@@ -5,6 +5,8 @@ import "react-notifications-component/dist/theme.css";
 import { addNotification } from "../../../utilities/index";
 import API from "../../../api/index";
 import "./CourseForm.scss";
+import CKEditor from '@ckeditor/ckeditor5-react';
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
```

Comment:
what classic is doing here

Reply:
Because ck5 gives you to choose from it's different editors and we have chosen classic editor.